### PR TITLE
Version 8.x-1.x

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

File: csv2sql.drush.inc [hunk 1]
```diff
@@ -200,12 +211,17 @@ function csv2sql_create_db($table_name, $header = array(), $drop_existing = TRUE
...
+    return Database::getConnection()
+      ->insert($table_name)
       ->fields($row)
       ->execute();
   }
   catch (Exception $exception) {
-    drush_log(format_string('Error in row #@row: @message', array('@row' => $row['_id'], '@message' => $exception->getMessage())), 'error');
+    $string = 'Error in row #@row: @message';
+    $args = array('@row' => $row['_id'], '@message' => $exception->getMessage());
+    $markup = new FormattableMarkup($string, $args);
```
Comment (amitaibu) [inline csv2sql.drush.inc]: `sprintf()`

**Comment by HelenaEksler** [comment]
@amitaibu Done with `sprintf()`.

**Comment by HelenaEksler** [comment]
As I see in other modules - we should simply publish the 8.x version as we have the 7.x to the Drupal.org and it will be downloaded with `drush dl csv2sql` for Drupal 8.

**Comment by amitaibu** [comment]
> drush dl csv2sql for Drupal 8.

but this downloads it to `~/.drush` -- so it doesn't have a Drupal context

**Comment by HelenaEksler** [comment]
Also now doesn't matter where I'm putting the repository with git clone - I should copy the file to the `~/.drush`, but run the command from inside the `web` directory.

**Comment by amitaibu** [comment]
you can maybe also put the file in the `drush` folder in the root of your repo

**Comment by amitaibu** [comment]
btw, congrats on your new commit access on d.o! :)

csv2sql___drupal_org

**Comment by amitaibu** [comment]
I've pushed the branch to d.o.

**Comment by amitaibu** [comment]
Thanks!

**Pull request closed without merging**